moped-database/migrations/1617934601367_alter_table_public_moped_categories_update_comment/up.sql

```diff
@@ -0,0 +1,8 @@
+COMMENT ON TABLE "public"."moped_categories" IS E'DELETE unless needed; categories are project types in IMPD';
+COMMENT ON TABLE "public"."moped_city_fiscal_years" IS E'DELETE';
```

Comment:
As a note to self: It looks like we need this table for the time being.

https://github.com/cityofaustin/atd-moped/blob/5400a1b79d4539340a234f0a7bf720ec2322bc37/moped-editor/src/views/projects/newProjectView/DefineProjectForm.js#L36